HardwareBeginner.md

```diff
@@ -0,0 +1,68 @@
+# **PART 2**
+
+This is the second step in the process towards automation!
+
+# (1) Select how far into automation you want to get
+
+**No matter the level of automation you need, start here!**
+
+Select your level of automation 
+
+| [Microcontroller](HardwareBeginner.md#2-microcontroller-hardware) | [Computer Controlled](HardwareBeginner.md#2-microcontroller-hardware) |
+| --- | --- |
+| Basic Den hosting | All microcontroller programs plus... | 
+| Day Skipping | Automated Dynamax Adventures! |
+| Egg Hatchers | Den Hosting with stats!  |
+| Unattended Shiny Hunts | Autonomous Shiny Hunts! |
```

Comment:
Maybe we should link to the program lists for each of these as well?

Also, the "Shiny Hunts" parts might be misleading for BDSP since it seems like there will be no MC shiny hunts for BDSP at all at this point.